future-apps/staking/contracts/Staking.sol

```diff
@@ -97,6 +104,10 @@ contract Staking is ERCStaking, AragonApp {
     }
   }
 
+  function lockIndefinitely(uint256 amount, address unlocker, bytes32 metadata, bytes data) public returns(uint256 lockId) {
```

Comment:
Is there any reason for adding a function for this? See a limited usecase and people can do it themselves anyway

Reply:
Well, I thought it may be convenient for the user to avoid the need of `uint8(TimeUnit.Seconds), MAX_UINT64`, but I can remove it if you don't think so.